src/components/FormAddBook.js

```diff
@@ -0,0 +1,63 @@
+import { React, useRef } from 'react';
+import { useDispatch } from 'react-redux';
+import { nanoid } from 'nanoid';
+import { addBookApi } from '../redux/books/books';
+import './FormAddBook.css';
+
+function FormAddBook() {
+  const titleInputRef = useRef();
+  const categoryInputRef = useRef();
+  const dispatch = useDispatch();
+
+  function submitBookToStore(event) {
+    event.preventDefault();
+    const newBook = {
+      item_id: nanoid(),
+      title: titleInputRef.current.value,
+      category: categoryInputRef.current.value,
+    };
+    dispatch(addBookApi(newBook));
+    titleInputRef.current.value = '';
+    categoryInputRef.current.value = '';
+  }
+
+  return (
+    <div className="add-book-container">
+      <h2 className="add-title">ADD NEW BOOK</h2>
+      <form onSubmit={submitBookToStore} className="flex justify-between">
+        <input
+          type="text"
+          name="name"
+          placeholder="Book Title"
+          id="title"
+          ref={titleInputRef}
+          required
+          className="input-title"
+        />
```

Comment:
- [ ] It's still not updating. Kindly recheck, confirm and update.

![image](https://user-images.githubusercontent.com/51296741/156849231-0b1a36f0-41bf-45d4-a231-2b0d945700e7.png)

Reply:
Can we get on the zoom call I show you from side

Reply:
@zairdon20 why don't you add the screenshot here? As a confirmation that everything works from your end.

Reply:
![image](https://user-images.githubusercontent.com/90151551/156870956-881c27d8-50bc-43af-9e1f-46458acec297.png)

Reply:
There is the image

Reply:
![Screenshot_React App -Josphat_5](https://user-images.githubusercontent.com/90151551/156873338-7662fc10-994b-45c8-8346-55a58ce38273.png)